docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/ShoppingCart.scala

```diff
@@ -97,9 +97,6 @@ object ShoppingCart {
   val empty: ShoppingCart = ShoppingCart(items = Map.empty)
   // #shopping-cart-empty-state
 
-  // #shopping-cart-entity-type-key
-  val typeKey: EntityTypeKey[Command] = EntityTypeKey[Command]("ShoppingCart")
-  // #shopping-cart-entity-type-key
```

Comment:
Hmmm, so `shopping-cart-entity-type-key` is no longer used in the `.md`? OK, docs validation will complain. :-)

Reply:
no, it won't complain. It's not being used. 

We are now using `#companion-with-type-key-and-factory` that has the entity key and the behavior together